Tests are failing

```
tempest:~/Code/Versal/highlightr-iframe $ grunt
Running "stylus:compile" (stylus) task
File assets/gadget.css created.

Running "mocha:test" (mocha) task
Testing: assets/test.html


  Highlightr prototype
    ✓ should have a prototype
    Highlightr instance
      ✓ not be editable initially
      ✓ should have a config
      ✓ should render
      1) should have an updated config
      2) should have a code snippet


  4 passing (313ms)
  2 failing

  1) Highlightr prototype Highlightr instance should have an updated config:
     AssertionError: expected undefined to equal 'default'
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:925
      at assertEqual (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:1402)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:3627
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/specs/highlightr_spec.js:50
      at callFn (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4338)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4331
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4728
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4819
      at next (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4653)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4663
      at next (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4601)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4630
      at timeslice (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:5761)

  2) Highlightr prototype Highlightr instance should have a code snippet:

      + expected - actual

      +1
      -0

      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:925
      at assertEqual (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:1402)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/chai.js:3627
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/specs/highlightr_spec.js:55
      at callFn (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4338)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4331
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4728
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4819
      at next (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4653)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4663
      at next (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4601)
      at file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:4630
      at timeslice (file:///Users/dylan/Code/Versal/highlightr-iframe/assets/lib/mocha.js:5761)



>> 2/6 tests failed (0.31s)
Warning: Task "mocha:test" failed. Use --force to continue.

Aborted due to warnings.
```

--- a/assets/specs/highlightr_spec.js
+++ b/assets/specs/highlightr_spec.js
@@ -26,18 +26,15 @@
       });
 
       it('should render', function(done) {
-        window.postMessage(
-          JSON.stringify({
-            event: 'attributesChanged',
-            data: {
-              code: 'function() { var whoop; }',
-              theme: 'default'
-              }
-          }),
-          '*'
-        );
+        window.postMessage({
+          event: 'attributesChanged',
+          data: {
+            code: 'function() { var whoop; }',
+            theme: 'default'
+            }
+        }, '*');
 
-        window.postMessage(JSON.stringify({ event: 'attached' }), '*');
+        window.postMessage({ event: 'attached' }, '*');
 
         this.timeout(500);
 

